CONTRIBUTING.rst

```diff
@@ -2,38 +2,56 @@
 Contributing to Klein
 =====================
 
-Klein is an open source project that welcomes contributions of all kinds coming from the community.
-
-These include:
+Klein is an open source project that welcomes contributions of all kinds coming from the community, including:
 
 - code patches,
 - `documentation <http://klein.readthedocs.org/>`_ improvements,
 - `bug reports <https://github.com/twisted/klein/issues>`_,
 - reviews for `contributed patches <https://github.com/twisted/klein/pulls>`_.
 
-Here's some suggestions to make the contributing process much smoother for you:
+
+Getting started
+===============
+
+Here is a list of shell commands that will install the dependencies of Klein, run the test suite, and compile the documentation.
+
+::
+   pip install --user -r requirements.txt
+   tox
```

Comment:
A bare `tox` runs *all* the environments, which is ~20 envs. I think instead we should recommend running just one (such as `-e py27-twcurrent` to run it on the most recent Twisted on Python 2.7) during development, and then the whole suite before issuing a PR.